resources/lang/en/message.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'title' => 'Laravel Quicktask Basic',
+    'english' => 'English',
+    'vietnamese' => 'Vietnamese',
+    'errors' => 'Whoops! Something went wrong!',
+    'task' => 'Task',
+    'edit' => 'Edit',
+    'task_list' => 'Task List',
+    'edit_task' => 'Edit Task',
+    'delete' => 'Delete',
+    'add_task' => 'Add Task',
+    'add' => 'Add',
+    'current_tasks' => 'Current Tasks',
+    'create_success' => 'Created successfully!',
+    'edit_success' => 'Updated successfully!',
+    'delete_success' => 'Deleted successfully!',
+    'fail' => 'Not found!',
+];
+?>
```

Comment:
```suggestion
<?php

return [
    'title' => 'Laravel Quicktask Basic',
    'english' => 'English',
    'vietnamese' => 'Vietnamese',
    'errors' => 'Whoops! Something went wrong!',
    'task' => 'Task',
    'edit' => 'Edit',
    'task_list' => 'Task List',
    'edit_task' => 'Edit Task',
    'delete' => 'Delete',
    'add_task' => 'Add Task',
    'add' => 'Add',
    'current_tasks' => 'Current Tasks',
    'create_success' => 'Created successfully!',
    'edit_success' => 'Updated successfully!',
    'delete_success' => 'Deleted successfully!',
    'fail' => 'Not found!',
];

?>
```
Hoặc không cần thẻ đóng cũng được